docs/sidebarsClassicML.ts

```diff
@@ -173,7 +173,7 @@ const sidebarsClassicML: SidebarsConfig = {
               items: [
                 {
                     type: 'doc',
-                    id: 'deep-learning/tensorflow/quickstart/index',
+                    id: 'deep-learning/tensorflow/quickstart/quickstart-tensorflow-ipynb',
```

Comment:
Could we fix this or remove it in this PR? I cannot build the website, it's hard to review 😅

Reply:
you need to generate the notebook file with the command `yarn convert-notebooks` in order to generate the .mdx equivalent of the jupyter notebook.